Basic-Video-Zoom/js/app.js

```diff
@@ -0,0 +1,128 @@
+import {
+  MediaProcessorConnector,
+} from "../node_modules/@vonage/media-processor/dist/media-processor.es.js";
+import { WorkerMediaProcessor } from "./media-processor-helper-worker.js";
+// import { GreyScaleTransformer } from './transformer.js';
+/* global OT API_KEY TOKEN SESSION_ID SAMPLE_SERVER_BASE_URL */
+/* global ResizeTransformer MediaProcessor MediaProcessorConnector */
+
+
+let apiKey;
+let sessionId;
+let token;
+
+const transformStream = async (publisher) => {
+  if (OT.hasMediaProcessorSupport()) {
+    const mediaProcessor = new WorkerMediaProcessor();
+    const mediaProcessorConnector = new MediaProcessorConnector(
+      mediaProcessor
+    );
+
+    publisher
+      .setVideoMediaProcessorConnector(mediaProcessorConnector)
+      .then(() => {
+        console.log("set connector");
```

Comment:
I believe console log statements like this one should be removed to clean up the logging